docs/getting-started-guides/ubuntu/troubleshooting.md

```diff
@@ -68,41 +68,11 @@ This will automatically ssh you to the easyrsa unit.
 
 ## Collecting debug information
 
-Sometimes it is useful to collect all the information from a node to share with a developer so problems can be identifying. This section will deal on how to use the debug action to collect this information. The debug action is only supported on `kubernetes-worker` nodes.
+Sometimes it is useful to collect all the information from a cluster to share with a developer so problems can be identified. This section will deal on how to use the [CDK Field Agent](https://github.com/juju-solutions/cdk-field-agent) to collect this information.
 
-    juju run-action kubernetes-worker/0 debug
+Download and execute the collect.py script from [CDK Field Agent](https://github.com/juju-solutions/cdk-field-agent) on a box that has Juju configured such that the current controller and model are pointed at your CDK deployment of interest.
```

Comment:
This would be better as two sentences:

> Download and execute the `collect.py` script from [CDK Field Agent](https://github.com/juju-solutions/cdk-field-agent) on a box that has Juju configured. The current controller and model point to your CDK deployment of interest.

Reply:
I agree this is poorly worded, but I don't think the solution is to break it into multiple sentences like that. The box needs Juju configured with the current controller and model point to the CDK to debug. I'll make a pass at trying to reword this so it makes sense.

Thanks for the input!